pom.xml

```diff
@@ -31,6 +36,24 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+
+      <dependency>
+        <groupId>com.github.javaparser</groupId>
+        <artifactId>javaparser-core</artifactId>
+        <version>${version.com.github.javaparser}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>org.assertj</groupId>
+        <artifactId>assertj-core</artifactId>
+        <version>${version.org.assertj}</version>
```

Comment:
This should be scope test, isnt it?

Reply:
No. That dependency is used in compile scope only by quarkus-openapi-generator-test-utils, which is then used in test scope by the other modules. The other modules don't have access to assertj in compile scope.

Reply:
Ok, thanks for the explanation